packages/wdio-mocha-framework/src/index.ts

```diff
@@ -202,8 +203,13 @@ class MochaAdapter {
 
         switch (hookName) {
         case 'beforeSuite':
+            this._suiteStartDate = Date.now()
+        // eslint-disable-next-line no-fallthrough
```

Comment:
Let's use break here rather than having to ignore eslint
```suggestion
        break;
```

Reply:
> Let's use break here rather than having to ignore eslint

@christian-bromann  the suite object is the same for before and after suite , thats why there was no break there . If we add the break the beforeSuite will have only type property. All other information will be removed

Reply:
@christian-bromann 

https://github.com/webdriverio/webdriverio/blob/08644c01ba6537cce209d57b4c534cba3f96c288/packages/wdio-mocha-framework/src/index.ts#L208-L209

The params.payload step is commond for both before and aftersuite , thats why we didnt add a break before

Reply:
so if i add break , the common statement of params.payload wont get executed . SHould i repeat that inside first case ?

Reply:
yeah .. my goal here is to get rid of this eslint comment .. at the end of the day we use it to follow its rules 😉